### DISCUSSION TITLE
Wrong file in releases? Version 5.2.2 != 5.3.0

### DISCUSSION THREAD

**Discussion by Colossus68** [LycheeOrg/Lychee#2418] [Bugs] [answered]
Was the wrong file accidentally used for the 5.3.0 release? Or was the merge not complete? 

version.md says 5.2.2 and the changes from #2413 are not present in the files resources/views/components/gallery/photo/sidebar.blade.php and resources/views/components/gallery/view/photo.blade.php.

Sorry, if I have done something wrong here ...

**Comment by ildyria** [comment]
You are right. Wrong file. Sorry for that.

**Reply by Colossus68** [reply]
No problem.

**Reply by ildyria** [reply, marked as answer]
Fixed.

**Reply by Colossus68** [reply]
Thank you very much! Greetings from Switzerland.
